Jenkinsfile

```diff
@@ -43,151 +44,67 @@ def build(Map m){
                 "subnet-341ff61f",
                 "us-east-1"
             ) {
-                stage('Checkout') {
-                    checkout scm
-                    dockerTag = readFile('mozilla-release/browser/config/version.txt').trim()
-                }
-                def baseImageName = "browser-f/android:${dockerTag}"
-                docker.withRegistry('https://141047255820.dkr.ecr.us-east-1.amazonaws.com') {
-                    def image = docker.image(baseImageName)
-                    image.pull()
-                    image.inside {
-                        stage('Build Cliqz React Native') {
-                            sh '''#!/bin/bash -l
-                                set -e
-                                set -x
-                                cd cliqz
-                                npm ci
-                                npm run build
-                            '''
-                        }
-
-                        stage("Build APK: ${flavorname}") {
-                            withEnv([
-                                "ANDROID_TARGET=${androidtarget}",
-                                "BRAND=${flavorname}",
-                                "CLIQZ_CHANNEL=MA99"
-                                ]) {
-                                sh '''#!/bin/bash -l
-                                    set -x
-                                    set -e
-                                    cp mozconfigs/jenkins.mozconfig mozilla-release/mozconfig
-                                    cd mozilla-release
-                                    ./mach clobber
-                                    ./mach build
-                                    for lang in `ls ../l10n/`; do
-                                        ./mach build chrome-$lang
-                                    done
-                                    ./mach package
-                                '''
+                try {
+                    stage('Checkout') {
+                        checkout scm
+                        dockerTag = readFile('mozilla-release/browser/config/version.txt').trim()
+                    }
+                    def baseImageName = "browser-f/android:${dockerTag}"
+                    docker.withRegistry('https://141047255820.dkr.ecr.us-east-1.amazonaws.com') {
+                        def image = docker.image(baseImageName)
+                        image.pull()
+                        image.inside {
+                            stage('Build Cliqz React Native') {
+                                cliqz.buildFreshTab("cliqz")
```

Comment:
Please, rename this: there is no FreshTab there and it may be confusing

Reply:
Done.